src/App.js

```diff
@@ -28,11 +28,13 @@ function App() {
   }, []);
 
   function deleteIssue(issueToBeDeleted) {
-    const filteredIssues = issues.filter((issue) => {
-      return issue !== issueToBeDeleted;
-    });
+    destroyIssue(issueToBeDeleted.id).then(() => {
+      const filteredIssues = issues.filter((issue) => {
+        return issue.id !== issueToBeDeleted.id;
```

Comment:
Because `issueToBeDeleted` isn't a reference to an issue object in the `issues` list (we fetched it from the API with `GET /issues/:id` in the last PR), deleting an issue won't remove that issue from the list. For example, if we try to delete issue with and `id` of 3, we have have 2 issues in our app with an `id` of 3. One came from `GET /issues` and the other came from `GET /issues/3`.